File: web/cgi-bin/python/mysql/database-insert.py
#!/usr/bin/python

import cgi, cgitb, MySQLdb, sys

#habilita debug no browser
cgitb.enable();

dados = cgi.FieldStorage() 

#bd = dados.getvalue('bd')
a0 = dados.getvalue('a0');
a1 = dados.getvalue('a1');
a2 = dados.getvalue('a2');
a3 = dados.getvalue('a3');
a4 = dados.getvalue('a4');
a5 = dados.getvalue('a5');

# Connect to MySQL database
db = MySQLdb.connection(host="127.0.0.1",user="root",passwd="root",db="sensores")

#db.query("INSERT INTO 'sensores'.'sensores'('data','a0','a1','a2','a3','a4','a5')VALUES(CURRENT_TIMESTAMP,?,?,?,?,?,?);",(a0,a1,a2,a3,a4,a5))

db.query("INSERT INTO `sensores`.`sensores`(`data`,`a0`,`a1`,`a2`,`a3`,`a4`,`a5`) VALUES(CURRENT_TIMESTAMP,1,3,4,5,6,7);")

#conn = sqlite3.connect(bd)
#c = conn.cursor()                            

#c.execute("INSERT INTO sensores VALUES (datetime('now','localtime'),?,?,?,?,?,?,'1','0')",(a0,a1,a2,a3,a4,a5))

#conn.commit()
#conn.close()

print "Status: 200 OK"
print "Content-Type: text/html"
print ""
print "%s" %(dados)

File: web/cgi-bin/php/mysql/database-insert.php
<?php
  
    $a0 = $_GET["a0"];
    $a1 = $_GET["a1"];
    $a2 = $_GET["a2"];
    $a3 = $_GET["a3"];
    $a4 = $_GET["a4"];
    $a5 = $_GET["a5"];
    $a6 = $_GET["a6"];
    $a7 = $_GET["a7"];
    $a8 = $_GET["a8"];
    $a9 = $_GET["a9"];
  try
  {
    
    //open the database
    $conn = new PDO('mysql:host=localhost;dbname=cer','root','root');

    $conn->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);

    //$db->exec("INSERT INTO usr (data, a0, a1, a2, a3, a4, a5)
    //VALUES (current_timestamp(),'$a0', '$a1', '$a2', '$a3', '$a4' , '$a5')");

    $stmt = $conn->prepare("INSERT INTO usrTemporario ( a0, a1, a2, a3, a4, a5, a6, a7, a8, a9)
    VALUES (:a0, :a1, :a2, :a3, :a4, :a5, :a6, :a7, :a8, :a9)");
    $stmt->bindParam(':a0', $a0);
    $stmt->bindParam(':a1', $a1);
    $stmt->bindParam(':a2', $a2);
    $stmt->bindParam(':a3', $a3);
    $stmt->bindParam(':a4', $a4);
    $stmt->bindParam(':a5', $a5);
    $stmt->bindParam(':a5', $a5);
    $stmt->bindParam(':a5', $a5);

    $stmt->execute();

    // close the database connection
    $db = NULL;
    echo "OK";
  }
  catch(PDOException $e)
  {
    print 'Exception : '.$e->getMessage();
  }  
?>
